Gems/PhysX/Code/Editor/ColliderCapsuleMode.h

```diff
@@ -28,28 +30,21 @@ namespace PhysX
         void Teardown(const AZ::EntityComponentIdPair& idPair) override;
         void ResetValues(const AZ::EntityComponentIdPair& idPair) override;
 
+        // CapsuleViewportEdit ...
+        const AZ::Transform GetCapsuleWorldTransform() const override;
+        const AZ::Transform GetCapsuleLocalTransform() const override;
+        const AZ::Vector3 GetCapsuleNonUniformScale() const override;
+        const float GetCapsuleRadius() const override;
+        const float GetCapsuleHeight() const override;
```

Comment:
These return types being `const` are restricting unnecessarily the copy the caller gets. Please remove the const in the return types (keep the function const ones).

Reply:
Removed the const.